examples/models/buttons.py

```diff
@@ -8,11 +8,11 @@
 radio buttons, dropdown menu, checkbox and radio with groups.
 
 .. bokeh-example-metadata::
-    :apis: bokeh.models.widgets.buttons.Button, bokeh.models.widgets.buttons.Dropdown, bokeh.models.widgets.buttons.Toggle, bokeh.models.widgets.groups.CheckBoxButtonGroup, bokeh.models.widgets.groups.CheckboxGroup, bokeh.models.widgets.groups.RadioButtonGroup, bokeh.models.widgets.groups.RadioGroup  # noqa: E501
+    :apis: bokeh.models.widgets.buttons.Button, bokeh.models.widgets.buttons.Dropdown, bokeh.models.widgets.buttons.Toggle, bokeh.models.widgets.groups.CheckBoxButtonGroup, bokeh.models.widgets.groups.CheckboxGroup, bokeh.models.widgets.groups.RadioButtonGroup, bokeh.models.widgets.groups.RadioGroup
```

Comment:
Can't this line be split up or alternatively non-qualified names resolved docs compilation time (afteral we have a flat namespace for pretty much all models)?

Reply:
Lots of things could be done, I haven't had time, and I was made aware that there may be an existing sphinx plugin to link all API functions in example code blocks automatically. My priority is evaluating that (when I have the time) and if that does not work, then making QOL improvements to our extension.

Reply:
Oh but to answer the primary question: no, at the moment it cannot be split up